.codeclimate.yml

```diff
@@ -1,4 +1,6 @@
 engines:
+  duplication:
+    enabled: false
```

Comment:
I wish CodeClimate let you specific which files to apply this to.  It's a good idea, but tests are going to be ***FULL*** of duplication, intentionally.

Reply:
yep, i totally agree -- i really wish we could keep this on and ignore it for test files...